src/frontend/Komponenter/Behandling/Vurdering/Delvilkår.tsx

```diff
@@ -18,28 +18,42 @@ const FontStyle = styled.div<{ italic: boolean }>`
     font-style: ${(props) => (props.italic ? 'italic' : 'normal')};
 `;
 
+const unntakFraHovedregelTekst = 'Er unntak fra hovedregelen oppfylt?';
+
 const Delvilkår: FC<Props> = ({ regel, vurdering, settVurdering }) => {
     const hjelpetekst = hjelpeTekstConfig[regel.regelId];
     return (
         <DelvilkårContainer>
             <RadioGroup legend={delvilkårTypeTilTekst[regel.regelId]} value={vurdering.svar || ''}>
-                {Object.keys(regel.svarMapping).map((svarId) => {
+                {Object.keys(regel.svarMapping).map((svarId, i) => {
                     const erTekstKursiv = tekstSkalKursiveres(svarId);
-
                     return (
-                        <Radio
-                            key={`${regel.regelId}_${svarId}`}
-                            name={`${regel.regelId}_${svarId}`}
-                            value={svarId}
-                            onChange={() =>
-                                settVurdering({
-                                    svar: svarId,
-                                    regelId: regel.regelId,
-                                })
-                            }
-                        >
-                            <FontStyle italic={erTekstKursiv}>{svarTypeTilTekst[svarId]}</FontStyle>
-                        </Radio>
+                        <>
+                            {delvilkårTypeTilTekst[regel.regelId] === unntakFraHovedregelTekst &&
```

Comment:
```suggestion
                            {regel.regelId === 'MEDLEMSKAP_UNNTAK' &&
```
Denne sjekken blir mer presis.

Slik det er gjort nå, så vil infoboksen også dukke opp for unntak for det neste inngangsvilkåret:
![Skjermbilde 2023-09-21 kl  13 16 14](https://github.com/navikt/familie-ef-sak-frontend/assets/32769446/a7ab3647-41f9-4824-8dbc-a9dc3b3348c3)